integration-tests/mongodb-rest-data-panache/src/main/java/io/quarkus/it/mongodb/rest/data/panache/TestController.java

```diff
@@ -0,0 +1,48 @@
+package io.quarkus.it.mongodb.rest.data.panache;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+@ApplicationScoped
+@Path("/test")
+public class TestController {
```

Comment:
We usually named REST endpoints `Resource` and not `Controller`

Reply:
Renamed to TestResource